src/raps/common.ts

```diff
@@ -46,20 +54,22 @@ export enum RapActionType {
   setNameENS = 'setNameENS',
 }
 
-export interface RapActionParameters {
+export interface RapEchangeActionParameters {
   amount?: string | null;
   assetToUnlock?: Asset;
   contractAddress?: string;
   inputAmount?: string | null;
   outputAmount?: string | null;
   tradeDetails?: Trade;
-  name?: string;
-  duration?: number;
-  rentPrice?: string;
-  recordKey?: string;
-  recordValue?: string;
-  salt?: string;
+}
+
+export interface RapENSActionParameters {
```

Comment:
I made a division in rap actions just to have types separated and easier to read